falcon_kit/bash.py

```diff
@@ -294,6 +294,8 @@ def script_run_consensus(config, db_fn, las_fn, out_file_bfn):
     params.update(locals())
     if config["falcon_sense_skip_contained"]:
         run_consensus = """LA4Falcon -H$CUTOFF -fso {db_fn} {las_fn} | """
+    elif config["falcon_greedy"]:
```

Comment:
I thought about the "greedy", however, my concern was "dose it reflect some 'greedy' algorithm?" I think the answer is "yes".